File: HRO/CMITDW Taal der Wiskunde/tentamen/Lua/o1.lua
-- -(i*2-1)^3 / i -+ ...
local function alg1(n)
	local sum = 0
	local op = true
	
	for i = 1, n do
		if op then
			sum = sum - ((i*2-1)^3) / i
		else
			sum = sum + ((i*2-1)^3) / i
		end
		
		op = not op -- toggle boolean
		print(i, sum)
	end
	
	return sum
end

print(alg1(600))

File: HRO/CMITDW Taal der Wiskunde/tentamen/o1.java
public class o1
{
	public static void main(String[] args)
	{
		System.out.println(func1(600)); 
	}
	
	// -(i*2-1)^3 / i -+ ...
	public static double func1(int n)
	{
		double sum = 0;
		boolean op = true;
		
		for (int i=1; i<=n; i++)
		{
			if (op)
				sum -= Math.pow(i*2-1, 3) / i;
			else
				sum += Math.pow(i*2-1, 3) / i;
			
			op = !op; // toggle boolean
			System.out.println(i+"\t"+sum);
		}
		
		return sum;
	}
}

